webapp/src/app/shared/table/table.component.ts

```diff
@@ -165,69 +181,123 @@ export class TableComponent implements OnInit, AfterViewInit, OnChanges, OnDestr
           },
     }
 
-    constructor(private assetGroupChangeService: AssetGroupObservableService) {}
+    constructor(
+        private ngZone: NgZone,
+        private windowExpansionService: WindowExpansionService,
+        ) {
+        this.windowExpansionService.getExpansionStatus().subscribe(()=>{
+            this.resetTableWidth();
+        })
+        this.dataSource = new TableDataSource(this.ngZone);
+        this.mainDataSource = new TableDataSource(this.ngZone);
+    }
+
+    scrollTableToPos(scrollPos){
+        this.viewport.scrollToOffset(scrollPos);
+    }
+
+    resetTableWidth(){
```

Comment:
@sidharthjain-zemoso this solution will introduce width flickering. Is there any solution to use css-magic with percent values?

Reply:
We need to find the root cause and try to revert this change. This was just a quick fix.